docs/markdown/Wrap-review-guidelines.md

```diff
@@ -26,6 +26,32 @@ $ cat ~/.config/mesonwrap.ini
 github_token = <github token>
 ```
 
+## Setting up the rewiew tooling
+
+The [Mesonwrap](https://github.com/mesonbuild/mesonwrap/) repository
+contains review tooling. It is used to do the actual review, but
+submitters can also use it to check their MRs. All issues reported by
+the tool must be fixed, so using the tool can speed up the review
+process considerably.
+
+The tool has some dependencies that are not available in all Linux
+distributions. Thus using a Python
+[Virtualenv](https://virtualenv.pypa.io/en/stable/). The tool can be
```

Comment:
Why virtualenv instead of the stdlib `python -m venv`?

Reply:
"Because it was there", mostly.